public/app/dashboard/profile.html

```diff
@@ -226,8 +226,7 @@ <h4 class="profile-activity-breakdown-header">Community Impacts</h4>
         <li class="assetlibrary-list-item list-inline col-xs-6 col-sm-4 col-md-3"
             data-ng-repeat="asset in user.assets.results">
           <div class="col-list-item-container">
-            <a id="user-asset-{{asset.id}}"
-              tool-href
+            <a tool-href
```

Comment:
Fyi, `data-ng-attr-id` would not work here. The way to put dynamic ids on these directive-driven tags is via directive itself. I.e., in `toolHrefDirective`.  Again, more trouble than it's worth.